pkg/server/endpoints/management.go

```diff
@@ -190,14 +190,16 @@ func (e *Endpoints) onboardHandler(c echo.Context) error {
 	return nil
 }
 
-func (e *Endpoints) validateToken(token string) (bool, error) {
-	t, err := e.DataStore.FetchAccessToken(context.TODO(), token)
+func (e *Endpoints) validateToken(c echo.Context, token string) (bool, error) {
```

Comment:
nit: for consistency
```suggestion
func (e *Endpoints) validateToken(ctx echo.Context, token string) (bool, error) {
```